contracts/KyberNetworkInterface.sol

```diff
@@ -12,9 +12,6 @@ interface KyberNetworkInterface {
     function enabled() public view returns(bool);
     function info(bytes32 id) public view returns(uint);
 
-    function searchBestRate(ERC20 src, ERC20 dest, uint srcAmount, bool usePermissionless) public view
```

Comment:
was this removed because it shouldn't be used externally? (since we have getExpectedRate)

Reply:
its removed since its not used. not because it shouldn't be used.